src/components/Container.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import NavTabs from "./NavTabs";
+import Fighters from "../pages/Fighters";
+import FrameData from "../pages/FrameData";
+import Blog from "../pages/Blog";
+import Twitter from "../pages/Twitter";
+import Footer from "./Footer";
+import Header from "./Header";
+
+export default function Container() {
+	const [currentPage, setCurrentPage] = useState("Fighters");
+
+	// This method is checking to see what the value of `currentPage` is. Depending on the value of currentPage, we return the corresponding component to render.
+	const renderPage = () => {
```

Comment:
This would be a great place for a vanilla js switch statement or react-router-dom Switch w/ Route component rendering.